haros/haros.py

```diff
@@ -67,6 +67,7 @@
 #   --home sets the HAROS directory (default "~/.haros")
 #   --config sets the location of the YAML file containing the configuration
 #            (default "~/.haros/configs.yaml")
+#   --xml causes HAROS to output JUnit XML format report files.
```

Comment:
I'll defer to @git-afsantos, but personally I feel `--xml` is a bit too generic: Haros could have some custom xml format that it could output, which would also be covered by an `--xml` command line option.

Perhaps `--junit-xml` or `--write-junit-xml` would be more appropriate.

Reply:
Thanks, good point. I changed it to ``--junit-xml-output`` - that should be clear and it's consistent with the internal variable / object naming which makes the code easier to understand.